### PR TITLE
Routes sync feature

### PR DESCRIPTION
**Pull request by adamgajzlerowicz**

### CODE REVIEW

File: frontend/src/App.vue [hunk 1]
```diff
@@ -54,11 +56,19 @@ export default {
...
+      sync: SYNC,
+      fetchUser: FETCH_USER,
+    }),
+  },
   created() {
     if (this.language) {
       /* eslint-disable-next-line no-underscore-dangle */
       this._i18n.locale = this.language;
     }
+    this.sync();
```
Comment (jacol12345) [inline frontend/src/App.vue]: `this[SYNC]()`?

File: frontend/src/store/actions.js [hunk 2]
```diff
@@ -64,4 +67,25 @@ export const actions = {
...
+  async [SYNC]({ dispatch, state, commit }) {
+    if (state.routes.length === 0 && state.user) {
+      dispatch(`${namespaces.drives}/${apiActions.fetchDrives}`);
+      return;
+    }
+
+    if (state.routes.length === 0 || !state.user || !navigator.onLine) {
+      return;
+    }
+
```
Comment (jacol12345) [inline frontend/src/store/actions.js]: Empty line but who cares.

File: frontend/src/store/actions.js [hunk 3]
```diff
@@ -64,4 +67,25 @@ export const actions = {
...
+
+    const { syncId, ...mappedRoute } = mapRoute(state.routes[0]);
+
+    try {
+      await post('drives', mappedRoute);
+      commit(SYNC_ITEM_SUCCESS, syncId);
+      dispatch(SYNC);
+    } catch (e) {
+      console.log(e);
```
Comment (jacol12345) [inline frontend/src/store/actions.js]: tutaj!

File: frontend/src/store/actions.js (hunk 2 above)
Comment (adamgajzlerowicz) [reply frontend/src/store/actions.js]: we can add this to linter if that is an error

File: frontend/src/App.vue (hunk 1 above)
Comment (adamgajzlerowicz) [reply frontend/src/App.vue]: nope. line 61 maps action to a name. in this case it's called sync

File: frontend/src/store/actions.js (hunk 3 above)
Comment (adamgajzlerowicz) [reply frontend/src/store/actions.js]: ok

File: frontend/src/store/actions.js (hunk 2 above)
Comment (adamgajzlerowicz) [reply frontend/src/store/actions.js]: but yeah - look weird

**Comment by jacol12345** [comment]
Please resolve conflict

**Pull request merged**